src/iop/colorbalance.c

```diff
@@ -272,10 +272,10 @@ static void add_preset(dt_iop_module_so_t *self, const char *name,
 void init_presets(dt_iop_module_so_t *self)
 {
   // these blobs were exported as dtstyle and copied from there:
-  add_preset(self, _("split-toning teal-orange (2nd instance)"),
+  add_preset(self, _("Split Toning teal-orange (2nd instance)"),
```

Comment:
I would probably standardize with "split-toning".

Changing the case here is not possible or we need to change all the case of all dt strings.

Reply:
See comment above.